File: 参考/YanJing-Manager/RegionAI-Node-Server/YanJing-Node-Server/src/api/router.ts
import { OpenAPIHono } from '@hono/zod-openapi'
import { router as userRouter } from '../controllers/UserController.js'
import { router as healthRouter } from '../controllers/HealthController.js'
import { captchaController } from '../controllers/CaptchaController.js'

const apiRouter = new OpenAPIHono()

// 装配到主 Router (类似 FastAPI 的 router.include_router)
apiRouter.route('/health', healthRouter)
apiRouter.route('/users', userRouter)
apiRouter.route('/captcha', captchaController)

export default apiRouter

File: 参考/YanJing-Manager/RegionAI-Node-Server/参考/Server/Python-FastAPI/app/api/router.py
from fastapi import APIRouter

from app.controllers import HealthController
from app.controllers import UserController
from app.controllers import PCmethods
from app.controllers import WeatherController

router = APIRouter()


router.include_router(HealthController.router, prefix="/api")
router.include_router(UserController.router, prefix="/api")
router.include_router(PCmethods.router, prefix="/api")
router.include_router(WeatherController.router, prefix="/api")
